framework/Cargo.toml

```diff
@@ -32,7 +32,7 @@ license = "GPL-3.0-or-later"
 keywords = ["cosmos", "cosmwasm", "framework"]
 
 [workspace.dependencies]
-cosmwasm-std = { version = "1.5.0", features = ["cosmwasm_1_2", "stargate"] }
```

Comment:
even if the stargate feature is enabled implicitly I'd still prefer having it enabled explicitly, no?

Reply:
It's enabled on proxy explicitly

Reply:
and ibc-client / host / manager?